frontend/test/cypress/integration/product-list/filters.spec.ts

```diff
@@ -54,12 +47,11 @@ describe('product list filters', () => {
          user.findByText(/item type:\s*cables/i).should('exist');
       });
 
-      waitForSearchCompletion();
-
       user
          .findByRole('button', { name: /collapse item type/i, expanded: true })
          .click(getVirtualListClickOptions());
 
+      user.wait('@search');
       user.wait(500);
```

Comment:
We should use the new constant `DEBOUNCE_MS` here as well

Reply:
Good question! It's because they have different meanings.. Debounce is for when I'm actually using debounce in the ui, here instead I'm just waiting an arbitrary amount of time, and I'm trying to make it shorter.. I'd like to improve this eventually